backoffice/pages/catalog/categories/create.tsx

```diff
@@ -6,13 +6,15 @@ import { createCategory, getCategories } from '../../../modules/catalog/services
 import { CATEGORIES_URL } from '../../../constants/Common';
 import { handleCreatingResponse } from '../../../modules/catalog/services/ResponseStatusHandlingService';
 import { useRouter } from 'next/router';
+import CategoryImage from '../../../modules/catalog/components/CategoryImage';
+import { SubmitHandler, useForm } from 'react-hook-form';
 
 const CategoryCreate: NextPage = () => {
   const router = useRouter();
+  const { handleSubmit, setValue } = useForm<Category>();
   var slugify = require('slugify');
   const [categories, setCategories] = useState<Category[]>([]);
-  const handleSubmit = async (event: any) => {
-    event.preventDefault();
+  const onHandleSubmit: SubmitHandler<Category> = async (data: any, event: any) => {
```

Comment:
Should refactor this function implemented.

For param 'data', it has a type Category because we call useForm as type Category in line 14.
Remove param 'event' and gets values from param 'data'.